workers/notifier/src/test/kotlin/NotifierRunnerTest.kt

```diff
@@ -62,6 +64,13 @@ class NotifierRunnerTest : WordSpec({
                     useSsl = false,
                     fromAddress = "no-reply@oss-review-toolkit.org"
                 )
+            ),
```

Comment:
Unfortunately, the test is not really meaningful - but this is out of scope of this PR.